miden-lib/src/transaction/errors.rs

```diff
@@ -10,26 +10,78 @@ use miden_objects::{accounts::AccountStorage, AssetError, Digest};
 pub enum TransactionKernelError {
     InvalidStorageSlotIndex(u64),
     MalformedAssetOnAccountVaultUpdate(AssetError),
+    MalformedRecipientData(Vec<Felt>),
+    MalformedAsset(AssetError),
     MissingStorageSlotValue(u8, String),
     UnknownAccountProcedure(Digest),
+    MalformedTag(Felt),
+    MalformedNoteType(NoteError),
+    MalformedAccountId(AccountError),
+    MalformedNoteInputs(NoteError),
+    MalformedNoteScript(Vec<Felt>),
+    MalformedNoteMetadata(NoteError),
 }
 
 impl fmt::Display for TransactionKernelError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Self::InvalidStorageSlotIndex(index) => {
+            TransactionKernelError::InvalidStorageSlotIndex(index) => {
```

Comment:
`Self::` confused the rust-analyzer and broke the suggested actions. I have done similar changes in other places, where `use TransactionKernelError::*` caused issues (namely, autocompletion is gone, actions don't work)